02_configure_host.sh

```diff
@@ -23,17 +23,8 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -i tripleo-quickstart-config/metalkube-inventory.ini \
     -b -vvv tripleo-quickstart-config/metalkube-setup-playbook.yml
 
-# Allow local non-root-user access to libvirt ref
-# https://github.com/openshift/installer/blob/master/docs/dev/libvirt-howto.md#make-sure-you-have-permissions-for-qemusystem
```

Comment:
For the avoidance of future confusion should we remove the old /etc/polkit-1/rules.d/80-libvirt.rules or just assume folks will be using a clean install?